open_schools_platform/parent_management/families/services.py

```diff
@@ -0,0 +1,29 @@
+from open_schools_platform.parent_management.families.models import Family
+from open_schools_platform.parent_management.parents.models import ParentProfile
+from open_schools_platform.student_management.student_profile.models import StudentProfile
+
+
+def create_family(name: str) -> Family:
+    family = Family.objects.create_family(
+        name=name,
+    )
+    return family
+
+
+def add_parents_to_family(family: Family, parent: ParentProfile):
```

Comment:
1. there should be list of parents in input parameters
2. we should rename service name

I think second variant suits